bcda/api/requests.go

```diff
@@ -514,17 +515,55 @@ func (h *Handler) bulkRequest(w http.ResponseWriter, r *http.Request, reqType se
 	}
 }
 
-func (h *Handler) validateRequest(resourceTypes []string) error {
+func (h *Handler) getResourceTypes(parameters middleware.RequestParameters, cmsID string) []string {
+	resourceTypes := parameters.ResourceTypes
+
+	// If caller does not supply resource types, we default to all supported resource types for the specific ACO
+	if len(resourceTypes) == 0 {
+		if acoConfig, found := h.Svc.GetACOConfigForID(cmsID); found {
+			if utils.ContainsString(acoConfig.Data, constants.Adjudicated) {
+				resourceTypes = append(resourceTypes, "Patient", "ExplanationOfBenefit", "Coverage")
+			}
+
+			if utils.ContainsString(acoConfig.Data, constants.PreAdjudicated) {
+				resourceTypes = append(resourceTypes, "Claim", "ClaimResponse")
+			}
+		}
+	}
+
+	return resourceTypes
+}
+
+func (h *Handler) validateRequest(resourceTypes []string, cmsID string) error {
 
 	for _, resourceType := range resourceTypes {
```

Comment:
`resourceType` is a bit confusing here, as it's referring to a string, rather than the ResourceType struct. Could we maybe use something like `resourceString` instead (and perhaps `resourceType` in place of `resource`)? Or on the flip side, maybe `ResourceType` could be `ResourceAccess`/`ResourceData` (or similar)?